src/main/java/org/fundacionjala/coding/yury/BanckOCR.java

```diff
@@ -0,0 +1,105 @@
+package org.fundacionjala.coding.yury;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * Created by YuryOrtuno on 5/22/2017.
+ */
+public class BanckOCR {
+    public static final int MAX_PASSWORD_SIZE = 27;
+    public static final int SPACE = 3;
+    public static final int LINE2 = 27;
+    public static final int LINE3 = 54;
+    public static final int MODULE = 11;
+    private static Map<String, String> map = new HashMap<>();
+
+    /**
+     * Constructed for default.
+     */
+    public BanckOCR() {
+        map.put(" _ | ||_|", "0");
+        map.put("     |  |", "1");
+        map.put(" _  _||_ ", "2");
+        map.put(" _  _| _|", "3");
+        map.put("   |_|  |", "4");
+        map.put(" _ |_  _|", "5");
+        map.put(" _ |_ |_|", "6");
+        map.put(" _   |  |", "7");
+        map.put(" _ |_||_|", "8");
+        map.put(" _ |_| _|", "9");
+    }
+
+    /**
+     * @param input String de number
+     * @return number
+     */
+    public String convert(String input) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (int i = 0; i < MAX_PASSWORD_SIZE; i = i + SPACE) {
+            StringBuilder stringBuilderAux = new StringBuilder();
+            stringBuilderAux.append(input.substring(i, i + SPACE));
+            stringBuilderAux.append(input.substring(i + LINE2, i + LINE2 + SPACE));
+            stringBuilderAux.append(input.substring(i + LINE3, i + LINE3 + SPACE));
+            String value;
+            value = map.get(stringBuilderAux.toString()) == null ? "?" : map.get(stringBuilderAux.toString());
+            stringBuilder.append(value);
+        }
+        return stringBuilder.toString();
+    }
+
+    /**
+     * @param input number
+     * @return sum number%11
+     */
+    public int calculateSum(String input) {
```

Comment:
This method should return if the number has a validCheckSum, that means a boolean